src/app/shared/components/archive-type-select/archive-type-select.component.ts

```diff
@@ -0,0 +1,33 @@
+/* format */
```

Comment:
We're only ever expecting to use this component in the onboarding flow, right? In that case, can we just include it in the `OnboardingModule` instead of the `SharedModule`? Ideally it wouldn't even need to be in `exports`, since the rest of the application doesn't even need to access this component.

Though if we know we want to use this in multiple places, the shared module is fine.

Reply:
I'm still leaning towards having this stay in the OnboardingModule for the time being; personally I want to try to prioritize having stronger boundaries in this codebase and as of right now there's no definite reason as to why the rest of the application that's importing the `SharedModule` should have access to this specific component. I think if we do want to use it later, than moving it back to the `SharedModule` should be pretty trivial.

This could be a question for the experience team though (pinging @k8lyn6 again), if they know we want to reuse this specific design later, then it can probably stay in the `SharedModule`.

Reply:
@meisekimiu I don't know for sure if/how we would reuse it in the future. I could see us reusing it, but I think for the time being your reasoning for it staying in OnboardingModule makes sense.